xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-slash/plugin.js

```diff
@@ -713,6 +713,13 @@
           iconClass: 'fa fa-image',
           description: editor.localization.get('xwiki-slash.action.img.hint'),
           outputHTML: '/img::'
+        }, {
+          group: 'Content',
+          id: 'icon',
+          name: editor.localization.get('xwiki-slash.action.icon.name'),
+          iconClass: 'fa fa-image',
```

Comment:
I'm wondering if we shouldn't use a different icon than the one used for Image quick action. There aren't any good options though. I can think only of https://fontawesome.com/v4/icon/file-image-o which is similar but different.

Reply:
I wondered too, but I thought it wasn't appropriate for an icon. Maybe do the other way around and change the image Quick Action's icon?

Reply:
> Maybe do the other way around and change the image Quick Action's icon?

There aren't many alternatives for the Image Quick Action icon either. Let's keep it like this for now.